images/ceph/Makefile

```diff
@@ -121,3 +123,15 @@ csv: $(OPERATOR_SDK) $(YQ) ## Generate a CSV file for OLM.
 
 csv-clean: $(OPERATOR_SDK) $(YQ) ## Remove existing OLM files.
 	@rm -fr ../../cluster/olm/ceph/deploy/* ../../cluster/olm/ceph/templates/*
+
+# list-image creates list of images for offline installation
+list-image:
+	@echo "producING list of images for offline installation";\
+	# remove the file if already exists
+	[ -f ../../cluster/examples/kubernetes/ceph/rook-ceph-image.txt ] && rm ../../cluster/examples/kubernetes/ceph/rook-ceph-image.txt;\
+	awk '/image:/ {print $2}' ../../cluster/examples/kubernetes/ceph/operator.yaml ../../cluster/examples/kubernetes/ceph/cluster.yaml | \
+	cut -d: -f2- |\
```

Comment:
I have to add this `cut -d: -f2- `because when running older command(`cut -d: -f2-`) from running from cli has no issue but when running from `make build` it generates in this format
```
          image: rook/ceph:master
    image: quay.io/ceph/ceph:v16.2.5
  # ROOK_CSI_CEPH_IMAGE: quay.io/cephcsi/cephcsi:v3.4.0
  # ROOK_CSI_REGISTRAR_IMAGE: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.2.0
  # ROOK_CSI_RESIZER_IMAGE: k8s.gcr.io/sig-storage/csi-resizer:v1.2.0
  # ROOK_CSI_PROVISIONER_IMAGE: k8s.gcr.io/sig-storage/csi-provisioner:v2.2.2
  # ROOK_CSI_SNAPSHOTTER_IMAGE: k8s.gcr.io/sig-storage/csi-snapshotter:v4.1.1
  # ROOK_CSI_ATTACHER_IMAGE: k8s.gcr.io/sig-storage/csi-attacher:v3.2.1
  # CSI_VOLUME_REPLICATION_IMAGE: quay.io/csiaddons/volumereplication-operator:v0.1.0

```